config.custom.js

```diff
@@ -2,17 +2,28 @@ const webpackConfig = require("./webpack.config");
 
 module.exports = (env, args) => {
     const config = {
-        PRINTER_NAME: "Original Prusa i3",
+        PRINTER_NAME: "Original Prusa MK3",
```

Comment:
Tohle je špatně. Ten název je: "Original Prusa i3 MK3S/+", problém je, že ten PrusaLink může běžet na MK2.5, MK3 i MK3S, což tam lze doplnit pouze live, proto to tam nedáváme.